### PR TITLE
Update azure-application-gateway.md

### PR DESCRIPTION
**Pull request by asukumar22**
Fixed grammar errors

### CODE REVIEW

**Comment by Court72** [comment]
@greg-lindsay

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by American-Dipper** [comment]
#assign:martinekuan
	
@MicrosoftDocs/cloud-architecture-content-team-pr-reviewers - Please review the proposed changes. If you want to move forward with the content, add a sign-off.
	
#label:"aq-pr-triaged"
#label:"pending-content-team/business-approval"
Cc: @MicrosoftDocs/public-repo-pr-review-team

File: well-architected/services/networking/azure-application-gateway.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ Azure Advisor helps you ensure and improve continuity of your business-critical
 
 ## Operational excellence
 
-Monitoring and diagnostics are crucial. Not only can you measure performance statistics but also use metrics troubleshoot and remediate issues quickly. We recommend you review the [Operational excellence design principles](../../devops/principles.md).
+Monitoring and diagnostics are crucial for ensuring operational excellence of your Application Gateway. Not only can you measure performance statistics but also use metrics to troubleshoot and remediate issues quickly. We recommend you review the [Operational excellence design principles](../../devops/principles.md).
```
Comment (ckittel) [inline well-architected/services/networking/azure-application-gateway.md:133]: Not just for the health operation of the gateway, but also for the workload that it's fronting.  Can you expand this thought a bit more please?

**Review by ckittel — CHANGES_REQUESTED** [review]

**Review by asukumar22 — COMMENTED** [review]
Updated the appropriate phrasing including backends.

**Review by ckittel — APPROVED** [review]

**Pull request merged**